client/css/layout.css

```diff
@@ -812,10 +812,18 @@ body.jsonEdit:not(.hitTest) .widget.foreign, body.jsonEdit:not(.hitTest):not(.sh
  --notUsable:true;
 }
 
-body:not(.jsonEdit) .widget.notUsable  > *{
+body:not(.jsonEdit) .widget.notUsable {
   pointer-events: none !important;
 }
 
+.widget.zoomyNotUsable {
+  --zoomyNotUsable:true;
+ }
+ 
+ body:not(.jsonEdit) .widget.zoomyNotUsable > * {
```

Comment:
Can't you do `.widget.zoomy.notUsable` instead and remove the hack in the widget class?

If not, the code should at least be moved to `zoomy`. Its `classes` function could do a replace on the result of `super.classes`.

Reply:
Fixed